common-api/src/main/java/org/pmiops/workbench/exceptions/ExceptionAdvice.java

```diff
@@ -41,14 +52,14 @@
     if (relevantError.getClass().getAnnotation(ResponseStatus.class) != null) {
       statusCode = relevantError.getClass().getAnnotation(ResponseStatus.class).value().value();
     }
-    if (relevantError instanceof WorkbenchException) {
+    if (relevantError instanceof DataBrowserException) {
```

Comment:
Noticed that you Changed WorkbenchException to DataBrowserException, everything still lives in a package called `org.pmiops.workbench.exceptions`. Should that be changed to `org.pmiops.databrowser.exceptions`?

Reply:
Actually, the whole package structure is still org.pmiops.worbench. So, left it as it is.